community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactions.java

```diff
@@ -301,10 +300,6 @@ KernelTransactionHandle createHandle( KernelTransactionImplementation tx )
 
     private void assertRunning()
     {
-        if ( availabilityGuard.isShutdown() )
```

Comment:
this check was done on this layer before to ensure that we hold newTransactionsLock when we doing so, now in cases when we block for various reasons on this lock for example and wake up, later on, we will allow the transaction to be started even db is already off do not think that is desired behaviour